elections/2021/TOC/README.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Knative TOC Election - 2021 Voters Guide"
+linkTitle: "2021 Voters Guide"
+weight: 30
+type: "docs"
+---
+
+# 2021 Voters Guide - Knative TOC Election
+
+## Purpose
+
+The role of this election is to fill out the one (1) seats due for election this year on the [Knative Technical Oversight Committee]. The elected
+member will serve a two (2) year term.
+
+## Background
+
+This election will shape the future of Knative as a community and project.
+While WGs help shape the individual direction of the project, the
+[TOC Charter] covers the technical direction, health of the project
+and community as a whole.
+
+## Eligibility
+
+Please refer to the [TOC Election Charter] for:
+
+- [Eligibility for candidacy]
+- [Eligibility for voting] 
+
+Eligible Voters list coming soon.
+
+### Schedule
+
+| Date         | Event                    |
+| ------------ | ------------------------ |
+| April 13     | Announcement of Election |
+| April 23     | All candidate nominations and voting exception forms due by 0000 UTC (5pm Pacific) |
+| April 27     | Election Begins via email ballots |
+| May 11       | Election Closes by 0000 UTC (5pm Pacific) |
+| May 13       | Announcement of Results |
+
+## Election Officers
+
+In an effort to run a fair and transparent election, the following people
+have been asked by the Steering Committee to run this election:
+
+- [Josh Berkus](https://github.com/jberkus)
+- [Dawn Foster](https://github.com/geekygirldawn)
+
+You can reach us by emailing elections@knative.team
+
+## Candidacy Process
+
+**Nomination**
+
+If you want to stand for election, open a PR against the
+[knative/community repository](https://github.com/knative/community) to include
+your candidate profile in the `/elections/2021/TOC` folder, with your name as the
```

Comment:
so ... if we wanna do this with Elekto, I've realized that we need to change the director structure.  It expects all election folders to be one level below the general /election/ folder.  So 2021-TOC/  instead of 2021/toc/

Reply:
@jberkus Done. I just changed the directory structure to match what Elekto expects to see.